src/components/Header/Header.tsx

```diff
@@ -1,14 +1,27 @@
 import React from "react";
 import PetsIcon from "@mui/icons-material/Pets";
+import { makeStyles } from "@mui/styles";
+import { Theme } from "@mui/material";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  SpalshDiv: {
+    display: "flex",
+  },
+  HeaderID: {
+    fontFamily: "Comfortaa,cursive",
+  },
+  PIcon: {
+    position: "relative",
+    marginTop: "7%",
+  },
+}));
+
 export default function Header() {
+  const HeaderClasses=useStyles();
   return (
-    <div id="HeaderID">
-      {/* <img src="/images/DogPaw2.png" alt="not found"/> */}
-      <PetsIcon
-        className="PIcon"
-        style={{ position: "absolute", left:"40%" }}
-      />
-      <h1 style={{ fontFamily: "Comfortaa,cursive", display:"flex", flexWrap:"wrap"  ,marginLeft:"23px" }}>uPet</h1>
+    <div className={HeaderClasses.SpalshDiv}>
+      <PetsIcon className={HeaderClasses.PIcon} />
```

Comment:
Yes it's right, Component name should always start with Capital letter, but change classes convention

Reply:
changed classes convention